jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -791,7 +805,7 @@ matrixJob('Cassandra-devbranch-artifacts') {
     description(jobDescription)
     concurrentBuild()
     axes {
-        jdk(jdkLabel,'jdk_11_latest')
+        jdk(jdk(jdks('cassandra-4.1', 'artifacts')))
```

Comment:
A little hack to get jdk 8/11?

Reply:
4.1, what about 4.0?

Reply:
it's this type of stuff that makes having the stage jobs in cassandra-builds and unversioned (and pre- and post-commit handled differently) a PITA